plugin/etcd/README.md

```diff
@@ -113,6 +113,15 @@ etcd skydns.local {
 ...
 ~~~
 
+You can connect to an etcd cluster protected by a password by specifiying the credential in the Corefile.
+
+~~~
+etcd skydns.local {
+    endpoint http://localhost:2379
+    credentials username password
```

Comment:
can you add this to the initial how-to-use the etcd plugin, with:

~~~
credentials USERNAME PASSWORD
~~~~
and explain it's use in the bullets below?

Reply:
ok, will do

Reply:
starting on line 30

Reply:
Done